<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>生成器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="top1" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="numLbl">
        <property name="text">
         <string>数目</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="top2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="sizeLbl">
        <property name="text">
         <string>尺寸</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox2">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>1080</number>
        </property>
        <property name="value">
         <number>224</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider2">
        <property name="maximum">
         <number>1080</number>
        </property>
        <property name="value">
         <number>224</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottom" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="arcBtn">
        <property name="text">
         <string>Arc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="circleBtn">
        <property name="text">
         <string>Circle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
